src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -82,7 +82,13 @@ Copyright (c) .NET Foundation. All rights reserved.
     -->
   <Target Name="CopyFilesToPublishDirectory"
           DependsOnTargets="_CopyResolvedFilesToPublishPreserveNewest;
-                            _CopyResolvedFilesToPublishAlways" />
+                            _CopyResolvedFilesToPublishAlways">
+
+    <!-- Ensure there is minimal verbosity output pointing to the publish directory and not just the
+         build step's minimal output. Otherwise there is no indication at minimal verbosity of where
+         the published assets were copied. -->
+    <Message Importance="High" Text="$(MSBuildProjectName) -&gt; $([System.IO.Path]::GetFullPath('$(PublishDir)'))" />
```

Comment:
Since there are other targets after this target:

                            GeneratePublishDependencyFile;
                            GeneratePublishRuntimeConfigurationFile

It feels like this message should get written out at the very end of publish.  Maybe making a new target that is "PublishCompleted" or something similar?